ze_graph_ext.h

```diff
@@ -767,6 +767,25 @@ typedef struct _ze_graph_dditable_ext_1_6_t
 
 } ze_graph_dditable_ext_1_6_t;
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Mutable command lists NPU specific flags and structures
+#define ZE_MUTABLE_COMMAND_EXP_FLAG_INFERENCE_EXECUTE 0x40
+#define ZE_STRUCTURE_TYPE_MUTABLE_INFERENCE_EXECUTE_EXP_DESC 0x00030001
+
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Mutable inference execute descriptor
+typedef struct _ze_mutable_inference_execute_exp_desc_t
+{
+    ze_structure_type_t stype;                          ///< [in] type of this structure
+    const void* pNext;                                  ///< [in][optional] must be null or a pointer to an extension-specific
+                                                        ///< structure (i.e. contains stype and pNext).
+    uint64_t commandId;                                 ///< [in] command identifier
+    ze_graph_handle_t hGraph;                           ///< [in] handle of the graph
+    ze_graph_profiling_query_handle_t hProfilingQuery;  ///< [in][optional] handle of profiling query
```

Comment:
During the first review I didn't notice that the structure. Compare to the on in [this slide](https://intel-my.sharepoint.com/:p:/r/personal/javier_e_martinez_intel_com/Documents/VPU/MeteorLake/RemoteTensors/NPUOVRemoteTensorSupport_20240412.pptx?d=w8d91c381df2d45b58962d113d1948605&csf=1&web=1&e=xXU4RY&nav=eyJzSWQiOjQxNiwiY0lkIjoxNDA3MzUwNzM5fQ), the graph handle should be encoded in the CommandID as it's tied to a specific Graph[Execute|Initialize]. I was expecting the struct would be the same as [ze_mutable_kernel_argument_exp_desc_t](https://spec.oneapi.io/level-zero/latest/core/api.html#ze-mutable-kernel-argument-exp-desc-t).